### ISSUE TITLE
I just posted an article but it says it was posted 8 hours ago.

### ISSUE THREAD

**Issue by peter-lawrey** [HubPress/hubpress.io#345]
It appears that the hub press thinks I posted at midnight.

**Comment by peter-lawrey** [comment]
On the home page, it states it was posted at

2016-03-22 00:00:00

**Comment by anthonny** [comment]
@peter-lawrey you are right, post are only based on the date, time is not take in account.
By default, the date is the date of the creation of the post.
You can specify another date with the attribute `:published_at: yyyy-mm-dd`

But with this attribute, you can use time: 
`:published_at: 2016-03-01 10:00:00` 

And the result in the URL is: 
https://demo-hubpress.github.io/2016/03/01/100000-Test-published.html
The name of the file is *100000-Test-published.html*

It is a typo not a feature.

**Comment by anthonny** [comment]
Thank you for your feedback ;) :beers:

**Comment by peter-lawrey** [comment]
I assume that the time is in GMT in that case?

**Comment by peter-lawrey** [comment]
I also assume that after a day it won't matter so much.

**Comment by peter-lawrey** [comment]
How can I set the time with out it appearing in the title.  

Note in your example you have https://demo-hubpress.github.io/2016/03/01/100000-Test-published.html  instead of https://demo-hubpress.github.io/2016/03/01/Test-published.html when you set the tag to 10:00:00
